src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md

```diff
@@ -169,6 +189,11 @@ appBuilder.Services.AddOptions<OtlpExporterOptions>()
   `OtlpExportProtocol.Grpc` and "localhost:4318" for
   `OtlpExportProtocol.HttpProtobuf`.
 
+> [!NOTE]
+> When using `OtlpExportProtocol.HttpProtobuf`, the full URL MUST be
+> provided, including the signal-specific path v1/{signal}. For example, for
+> traces, the full URL will look like <http://your-custom-endpoint/v1/traces>.
```

Comment:
This is an invalid link (we don't expect folks to click it), change to backquotes.